tests.html

```diff
@@ -82,6 +82,29 @@
 				});
 				
 				startSection("Executive class"); // ===========================
+
+				runTest("Check for upper level eligibility ", () => {
```

Comment:
This is an example for a plan where ULE is satisfied right? If so there should also be a test with a plan where ULE is not satisfied.